### PR TITLE
Option to plot only pyscal data in `RelativePermeability` plugin

### LINKED ISSUE #866: Possibility to run `RelativePermeability` plugin only based on pyscal input file
Currently we support the use of a pyscal input file (with opt/base/high) to be used as a "SCAL recommendation". 
There is a request to have an option to drop the currently mandatory ensemble data inpu, and plot only the curves made by pyscal based on the input file. Considering we already support the data input, the feature is likely to be low hanging fruit.

### PR DESCRIPTION
**Pull request by asnyv**
*Insert a description of your pull request (PR) here, and check off the boxes below when they are done.*

---

### Contributor checklist

- [X] :tada: This PR closes #866.
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [X] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

**Comment by asnyv** [comment]
WLF will have to be next iteration. Need to get this out for testing by some SCAL specific users.

File: webviz_subsurface/_utils/colors.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from typing import Tuple
 
 
-class StandardColors(Enum):
+class StandardColors(str, Enum):
```
Comment (anders-kiaer) [inline webviz_subsurface/_utils/colors.py:6]: Should we here instead now use https://github.com/equinor/webviz-subsurface/blob/7c35594495cebb4d57584e7c2f8b347751b9f778/webviz_subsurface/plugins/_group_tree/_types.py#L1?

File: webviz_subsurface/plugins/_relative_permeability.py [hunk 2]
```diff
@@ -700,37 +821,157 @@ def add_realization_traces(df, color_by, curves, sataxis, colors, nplots):
...
+            #                 "color": "black",
+            #             },
+            #             "line": {"dash": linestyle.get(case, "solid")},
+            #             "showlegend": curve_no == 0 and satnum_no == 0 and case_no == 0,
+            #         }
+            #         for satnum_no, (satnum, satnum_df) in enumerate(df.groupby("SATNUM"))
+            #         for case_no, (case, case_df) in enumerate(satnum_df.groupby("CASE"))
+            #     ]
+            # )
+            ##
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_relative_permeability.py:860]: Add a comment in the comment on when this (potentially) will be needed again? If not - remove?

**Review by anders-kiaer — COMMENTED** [review]
👍 LGTM - a couple of small suggestions only. Add a changelog entry?

File: webviz_subsurface/plugins/_relative_permeability.py (hunk 2 above)
Comment (asnyv) [reply webviz_subsurface/plugins/_relative_permeability.py:860]: haha, yeah that should have been removed...

File: webviz_subsurface/_utils/colors.py (hunk 1 above)
Comment (asnyv) [reply webviz_subsurface/_utils/colors.py:6]: Thanks, didn't know it existed. Will test :)

Comment (anders-kiaer) [reply webviz_subsurface/_utils/colors.py:6]: 👍 It is quite new. The idea is that it then should be hopefully easier to deprecate + start using std.lib https://docs.python.org/3/library/enum.html#enum.StrEnum when `python lower bound >= 3.11`

**Review by anders-kiaer — APPROVED** [review]
🚀

**Pull request merged**